tests/testClassifier.py

```diff
@@ -38,8 +39,10 @@ def setup_module(module):
 class TestClassifier(lsst.utils.tests.TestCase):
 
     def setUp(self):
-        filename = os.path.join("tests", "testfiles", "reader_test.log")
-        reader = Reader(filename)
+        pkg = eups.productDir("ctrl_stats")
+        metrics = os.path.join(pkg, "tests", "testfiles", "test.metrics")
```

Comment:
I don't think you should be using the `ctrl_stats` location at all. I think you should be locating these files relative to the test file itself. Use `os.path.abspath(os.path.dirname(__file__))` to obtain the directory and that would allow you to reliably locate the `testfiles` directory without relying on EUPS or any environment variable.